test/connectionconfig.test.ts

```diff
@@ -36,22 +28,28 @@ const validJson =
 `;
 
 suite('ConnectionConfig tests', () => {
-    test('error message is shown when reading corrupt config file', () => {
-        let bufferMock = TypeMoq.Mock.ofType(Buffer, TypeMoq.MockBehavior.Loose, 0);
-        bufferMock.setup(x => x.toString()).returns(() => corruptJson);
+    // TODO #930 handle case where mssql.connections section of the settings file is corrupt
+    // test('error message is shown when reading corrupt config file', () => {
```

Comment:
Should this be commented out or removed? Is there any plan to add it back in the future?